ts/masters-es5/highcharts.src.ts

```diff
@@ -24,15 +24,42 @@ if (MSPointer.isRequired()) {
     MSPointer.compose(G.Chart);
 }
 if (!Array.prototype.includes) {
-    Array.prototype.includes = function <T>(
+    // eslint-disable-next-line no-extend-native
+    Array.prototype.includes = function <T> (
         searchElement: T,
         fromIndex?: number
-    ) {
+    ): boolean {
         return this.indexOf(searchElement, fromIndex) > -1;
     };
 }
+if (!Array.prototype.find) {
+    // eslint-disable-next-line no-extend-native
+    Array.prototype.find = function <T> (
+        predicate: (search: T, index: number, array: any[]) => boolean
+    ): T | undefined {
+        if (this === null) {
+            throw new TypeError(
+                'Array.prototype.find called on null or undefined'
+            );
+        }
+        if (typeof predicate !== 'function') {
+            throw new TypeError('predicate must be a function');
+        }
+        const list = Object(this),
+            length = list.length >>> 0,
+            thisArg = arguments[1];
```

Comment:
= scope parameter. This should go into the `find` parameters as the optional second, see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#thisarg).